scripts/clean-all.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-npm install lerna \
-lerna clean --yes \
-rm -rf node_modules
```

Comment:
I added it long ago, and someone has adapted it to the current structure. Obsolete.